service/routes.py

```diff
@@ -65,16 +65,16 @@ def index():
             required=True, description="The user_id of the relevant user"
         ),
         # credit card fields
-        "first_name": fields.String(required=False),
-        "last_name": fields.String(required=False),
-        "card_number": fields.String(required=False),
-        "expiry_month": fields.Integer(required=False),
-        "expiry_year": fields.Integer(required=False),
-        "security_code": fields.String(required=False),
-        "billing_address": fields.String(required=False),
-        "zip_code": fields.String(required=False),
+        "first_name": fields.String(required=False, description="The first name on the credit card"),
+        "last_name": fields.String(required=False, description="The last name on the credit card"),
+        "card_number": fields.String(required=False, description="The card number on the credit card"),
+        "expiry_month": fields.Integer(required=False, description="The expected expiration month of the credit card"),
+        "expiry_year": fields.Integer(required=False, description="The expected expiration year of the credit card"),
+        "security_code": fields.String(required=False, description="The security code on the credit card"),
+        "billing_address": fields.String(required=False, description="The billing address of the credit card"),
+        "zip_code": fields.String(required=False, description="The zip code used along with the address for the credit card"),
         # paypal fields
-        "email": fields.String(required=False),
+        "email": fields.String(required=False, description="The email address the user wish to receive info for the paypal"),
```

Comment:
```suggestion
        "email": fields.String(required=False, description="Email address that links to a PayPal account"),
```
maybe better description?